src/aggregate/functions/string.c

```diff
@@ -328,6 +328,28 @@ static int stringfunc_startswith(ExprEval *ctx, RSValue *result, RSValue **argv,
   return EXPR_EVAL_OK;
 }
 
+static int stringfunc_contains(ExprEval *ctx, RSValue *result, RSValue **argv, size_t argc, QueryError *err) {
+  VALIDATE_ARGS("contains", 2, 2, err);
+  VALIDATE_ARG_ISSTRING("contains", argv, 0);
+  VALIDATE_ARG_ISSTRING("contains ", argv, 1);
+
+  RSValue *str = RSValue_Dereference(argv[0]);
+  RSValue *pref = RSValue_Dereference(argv[1]);
+
+  char *p_str = (char *)RSValue_StringPtrLen(str, NULL);
+  size_t n;
+  char *p_pref = (char *)RSValue_StringPtrLen(pref, &n);
```

Comment:
We can use `const char *` since `strstr` takes const.
Also pass NULL instead of `&n` which isn't used.